projects/laji/src/app/shared-modules/select/select/select.component.ts

```diff
@@ -35,13 +35,17 @@ export class SelectComponent implements OnInit, OnChanges, OnDestroy {
   @Input() filterPlaceHolder = 'Search...';
   @Input() useFilter = true;
   @Input() selected: string[] = [];
+  @Input() selectedGlobalSubCategories: {[key: string]: SelectOptions[]} = {};
   @Input() open = false;
   @Input() disabled = false;
   @Input() outputOnlyId = false;
   @Output() selectedChange = new EventEmitter<string[]|string>();
   @Input() multiple = true;
   @Input() info: string;
   @Input() loading = false;
+  @Input() subCategories = [];
```

Comment:
This is used at least in the view to decide the class at least but maybe this could be changed to more generic `@Input() optionClass` and `@Input() optionContainerClass`. Is there really a need for any of the subCategories in this context?